pkg/skaffold/constants/constants.go

```diff
@@ -62,6 +62,7 @@ const (
 	DefaultCloudBuildMavenImage  = "gcr.io/cloud-builders/mvn"
 	DefaultCloudBuildGradleImage = "gcr.io/cloud-builders/gradle"
 	DefaultCloudBuildKanikoImage = "gcr.io/kaniko-project/executor"
+	DefaultCloudBuildPackImage   = "gcr.io/k8s-skaffold/pack"
```

Comment:
nit: can we move this gcb/buildpacks.go as a  local constant if there are cyclic dependencies issues?

Reply:
I'll move all those default image names next to where they are used in another PR